client/src/components/LoginModal.vue

```diff
@@ -259,11 +259,6 @@ async function emailUser() {
     try {
       if ((await userStore.sendResetEmail(form.value.email)) === 200) {
         errors.value.email = ''
-        notify({
-          title: 'Reset Email',
-          type: 'success',
-          text: 'Reset Email Sent'
-        })
```

Comment:
why has this been removed?

Reply:
I removed it because it was saying reset email sent regardless of whether an email was being sent or not. For security reasons, the backend gives the same response when the email isn't found in the database, and when the email is found in the database. I guess we can keep the alert, but the message should be something like "If a user matching that email exists, a reset email was sent"

Reply:
is there another place the notification can go?

Reply:
> A place other than an alert you mean?

dont worry ignore this.

Reply:
Yeah it's an easy change. Jasper and I made it so that there were identical responses so that malicious actors couldn't determine whether an email address had an account or not, but if you reckon that is less important than direct feedback to the user I'll change it

Reply:
Hmm, I think malicious actors can tell if an email is registered anyway by trying it in the signup modal. So maybe that above security consideration is futile anyway

Reply:
Yeah its probably not needed. They would need access to the emails password to get into the website anyway.

Reply:
Should be done now